realm/realm-library/src/androidTest/java/io/realm/ManagedRealmCollectionTests.java

```diff
@@ -760,13 +760,16 @@ public void mutableMethodsOutsideTransactions() {
                     case REMOVE_ALL:
                     case RETAIN_ALL:
                         expected = UnsupportedOperationException.class;
+                        break;
+                    default:
```

Comment:
The `case`s in this `switch` are not covering all values in the `Enum` and that causes some warnings of ErrorProne.
This `default` means all other `case`s can be ignored.